.github/workflows/test.yml

```diff
@@ -32,7 +39,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install -r requirements.txt
-          pip install "Django~=${{ matrix.django-version }}.0" .
+          pip install "django~=${{ matrix.django-version }}.0a1" .
```

Comment:
Why the `0a1` bit? Won't that make this install alpha releases?

Reply:
Ah. I hadn't realised 5.2 was an alpha. I don't think we should be doing this yet, as we can't guarantee just because we support a 5.2 alpha that we'll definitely work with the released version. Happy to merge this (without the alpha install change) the moment it's released, but not before.

Reply:
Okay, fair enough. At least I know dj-database-url works without any changes.